From ba1c5fe53991e906a4a08f3fff70c1978bdff650 Mon Sep 17 00:00:00 2001
From: ozeng <oak.zeng@amd.com>
Date: Tue, 6 Jun 2017 10:53:28 -0500
Subject: [PATCH 2216/2500] drm/amdgpu: Reserve 0-3 invalidation register sets
 for none-amdgpu usages

The latest engine allocation is:
Engine 0, 1: idle
Engine 2, 3: firmware
Engine 4~12: amdgpu ring
Engine 13~15: idle
Engine 16: kfd tlb invalidation
Engine 17: Gart flushes

Change-Id: I9a32cbcb133c07d7efad1a7bd07c6e2098c747e7
Signed-off-by: Oak Zeng <Oak.Zeng@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c b/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c
index 9a29aa3..54ecbe2 100644
--- a/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c
+++ b/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c
@@ -392,7 +392,7 @@ static int gmc_v9_0_early_init(void *handle)
 static int gmc_v9_0_late_init(void *handle)
 {
 	struct amdgpu_device *adev = (struct amdgpu_device *)handle;
-	unsigned vm_inv_eng[AMDGPU_MAX_VMHUBS] = { 3, 3 };
+	unsigned vm_inv_eng[AMDGPU_MAX_VMHUBS] = { 4, 4 };
 	unsigned i;
 
 	for(i = 0; i < adev->num_rings; ++i) {
-- 
2.7.4

